source/guides/1.0/getting-started.md

```diff
@@ -410,7 +410,7 @@ end
 ```
 
 Hanami provides a DSL to describe changes to our database schema. You can read more
-about how migrations work in the [migrations' guide](/guides/1.0/migrations/overview).
+about how migrations work in the [Migrations Guide](/guides/1.0/migrations/overview).
```

Comment:
here "migrations guide" as an entity, the migrations themselves don't have "guides" so it looks a bit funny to say "migrations' guide" :D :D or even "migration's guide" for one migration.